source/amazon/services/supported-services/index.rst

```diff
@@ -46,6 +46,9 @@ The next table contains the more relevant information about configuring each ser
 | Cisco        | :ref:`Umbrella <cisco_umbrella>`                    | bucket                | cisco_umbrella | <bucket_name>/<prefix>/<year>-<month>-<day>                                                                      |
 +--------------+-----------------------------------------------------+-----------------------+----------------+------------------------------------------------------------------------------------------------------------------+
 
+.. [#path]
```

Comment:
Remove this and instead expand the paragraph before the table, adding a similar explanation for the path column